sympy/printing/str.py

```diff
@@ -706,6 +707,8 @@ def _print_Complement(self, expr):
         return r' \ '.join(self._print(set) for set in expr.args)
 
     def _print_Quantity(self, expr):
+        if self._settings.get("abbrev", True):
```

Comment:
The second parameter should be the default value.

Reply:
I made such a blunder still all the test passed. :)